### PR TITLE
Update readme with first time setup

### PR DESCRIPTION
**Pull request by Mikulas**
Took me longer to setup from scratch then I care to admit :grimacing:

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,3 +2,23 @@ Nextras ORM Plugin for PhpStorm
...
 Provides support for [Nextras ORM](https://github.com/nextras/orm) � an ORM library for PHP.
+
+## Contributing
+
+### Provisioning
+
+- install [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/)
+- install PHP plugin
+- Import Project, Create project from existing sources, keep defaults (seemingly empty project will be created)
+- File, Project Structure, Modules, click +, New Module
```
Comment (hrach) [inline readme.md]: line 13--17 seems pretty weird. maybe we could fix it by commiting some part of project.

Comment (Mikulas) [inline readme.md]: Yeah it's pretty hardcore setup. Should I push `.idea`?

Comment (hrach) [inline readme.md]: I have read some article in official what to push from .idea, you should commit the whole .idea.

File: .idea/libraries/php_openapi.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<component name="libraryTable">
+  <library name="php-openapi">
+    <CLASSES>
+      <root url="jar://$USER_HOME$/Applications/PhpStorm EAP.app/Contents/plugins/php/lib/php-openapi.jar!/" />
+      <root url="jar://$USER_HOME$/Applications/PhpStorm EAP.app/Contents/plugins/php/lib/php.jar!/" />
```
Comment (Mikulas) [inline .idea/libraries/php_openapi.xml]: This is the only problematic part. Other paths are relative, but this presumes OS X, EAP, and installation to `~/Applications` (which is not even default).

Maybe there could be multiple files for Win, OS X, …?

File: readme.md (hunk 1 above)
Comment (Mikulas) [inline readme.md]: for the reference https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

File: .idea/libraries/php_openapi.xml (hunk 2 above)
Comment (hrach) [inline .idea/libraries/php_openapi.xml]: I suggest ignoring libraries/* and leave this steps in manual.

**Comment by Mikulas** [comment]
rebased and tested, seems to work

**Comment by hrach** [comment]
Thank you!

**Pull request merged**